src/pages/profile/components/ProfileCarousel.tsx

```diff
@@ -5,32 +5,74 @@ import {
 } from './ProfileCarousel.style';
 import { useCarousel } from '../hooks/useCarousel';
 import { selectedTabIndexState } from '../states/selectedTabIndex';
-import { FollowUsers } from '@pages/profile/components';
+import { FollowUsers, MeditationInfo } from '@pages/profile/components';
+import { TabItems } from '../utils/createTabItems';
+import { PROFILE_TABS } from '../constants/profileTabs';
+import { Follow } from '@/types/Follow';
 
 interface ProfileCarouselProps {
-  totalIndex: number;
+  tabItems: TabItems;
+  fullName: string;
 }
 
-const ProfileCarousel = ({ totalIndex }: ProfileCarouselProps) => {
+const ProfileCarousel = ({ tabItems, fullName }: ProfileCarouselProps) => {
+  const { MEDITATION, FOLLOWING, FOLLOWER, INFO } = PROFILE_TABS;
+
   const [selectedTabIndex, setSelectedTabIndex] = useRecoilState(
     selectedTabIndexState
   );
   const [carouselRef] = useCarousel(
     selectedTabIndex,
     setSelectedTabIndex,
-    totalIndex
+    Object.keys(tabItems).length - 1
   );
 
   return (
     <ProfileCarouselContainer ref={carouselRef}>
-      <ProfileCarouselItem>0</ProfileCarouselItem>
-      <ProfileCarouselItem>
-        <FollowUsers following={true}></FollowUsers>
-      </ProfileCarouselItem>
-      <ProfileCarouselItem>
-        <FollowUsers following={false}></FollowUsers>
-      </ProfileCarouselItem>
-      <ProfileCarouselItem>4</ProfileCarouselItem>
+      {Object.entries(tabItems).map(([label, tabItem], index) => {
+        switch (label) {
+          case MEDITATION:
+            return (
+              <ProfileCarouselItem key={index}>
+                {tabItem.value}
+              </ProfileCarouselItem>
+            );
+          case FOLLOWING:
+            return (
+              <ProfileCarouselItem key={index}>
+                <FollowUsers
+                  following={true}
+                  data={tabItem.data as Follow[]}
+                />
+              </ProfileCarouselItem>
+            );
+          case FOLLOWER:
+            return (
+              <ProfileCarouselItem key={index}>
+                <FollowUsers
+                  following={false}
+                  data={tabItem.data as Follow[]}
+                />
+              </ProfileCarouselItem>
+            );
+          case INFO:
+            return (
+              <ProfileCarouselItem key={index}>
+                <MeditationInfo
+                  data={tabItem.data as number[]}
+                  fullName={fullName}
+                />
+                {tabItem.value}
+              </ProfileCarouselItem>
+            );
+          default:
+            return (
+              <ProfileCarouselItem key={index}>
+                {tabItem.value}
+              </ProfileCarouselItem>
+            );
+        }
+      })}
```

Comment:
이런 식으로 함수로 분리하면 조금 더 깔끔해 보일거 같습니다.

```suggestion
 {Object.entries(tabItems).map(([label, tabItem], index) => {
            return (
              <ProfileCarouselItem key={index}>
                {carouseLabelChecker(label)}
              </ProfileCarouselItem>
            );
        }
      })}

const carouseLabelChecker = (label) => {
  switch (label)
    case FOLLOWING:
      return (
        <FollowUsers 
          following={true}
          data={tabItem.data as Follow[]}
        />
      )

}
```